src/utils/presenter.ts

```diff
@@ -13,6 +15,14 @@ export const getPresenterPage = (req: Request, res: Response, next: NextFunction
     const appData: ApplicationData = getApplicationData(req.session);
     const presenter = appData[PresenterKey];
 
+    if (isRemoveJourney(req)){
+      return res.render(config.UPDATE_PRESENTER_PAGE, {
+        journey: config.JourneyType.remove,
+        backLinkUrl: `${config.UPDATE_FILING_DATE_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`,
```

Comment:
the backlink is going to the wrong page, if you look at the mural page, the update-filing-date page isn't part of the remove journey. It should go to the ${config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}

Reply:
Can you make sure that you test your changes locally before raising a PR.

Reply:
tested